timescaledb/how-to-guides/backup-and-restore/index.md

```diff
@@ -1,10 +1,14 @@
 # Backup and restore
 
-TimescaleDB takes advantage of the reliable backup and restore functionality provided by PostgreSQL. There are a few different mechanisms you can use to backup your TimescaleDB database:
+TimescaleDB takes advantage of the reliable backup and restore functionality
+provided by PostgreSQL. There are a few different mechanisms you can use to
+backup your TimescaleDB database:
+*   The [timescaledb-backup][timescaledb-backup] tool.
```

Comment:
I'd actually have this be a subset of logical backups so: 
Logical backups with either the [timescaledb-backup][timescaledb-backup] tool or with the [standard `pg_dump` / `pg_restore`][logical-backups] tooling